charts/dealbot/templates/deployment.yaml

```diff
@@ -0,0 +1,145 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-dealbot-controller
+  labels:
+    app: dealbot-controller
+{{- if .Values.labels }}
+{{ toYaml .Values.labels | indent 4 }}
+{{- end }}
+
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: dealbot-controller
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: dealbot-controller
+        chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+        heritage: {{ .Release.Service }}
+        release: {{ .Release.Name }}
+    spec:
+      volumes:
+      - name: config-volume
+        configMap:
+          name: {{ .Release.Name }}-{{ .Values.controller.config }}
+          items:
+          - key: config.toml
+            path: config.toml
+      containers:
+      - name: dealbot-controller
+        image: "{{ .Values.image.repo }}:{{ .Values.image.tag }}"
+        imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+        ports:
+          - containerPort: 9991
+            name: metrics
+          - containerPort: 8764
+            name: dealbot
+        command: [ "controller" ]
+        args: [ "--configpath", "/config.toml" ]
+        env: []
+        volumeMounts:
+        - name: config-volume
+          mountPath: /config.toml
+          subPath: config.toml
+          readOnly: true
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
```

Comment:
https://github.com/filecoin-project/dealbot/pull/37 will allow settings to be specified either directly as args or as env vars, and removes need for this